package-lock.json

```diff
@@ -1,222 +1,8 @@
 {
   "name": "website-welcome",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
-    "": {
-      "name": "website-welcome",
-      "version": "1.0.0",
-      "license": "ISC",
-      "devDependencies": {
-        "pre-commit": "^1.2.2",
-        "prettier": "2.2.1"
-      }
-    },
-    "node_modules/buffer-from": {
-      "version": "1.1.1",
-      "resolved": "https://registry.npmjs.org/buffer-from/-/buffer-from-1.1.1.tgz",
-      "integrity": "sha512-MQcXEUbCKtEo7bhqEs6560Hyd4XaovZlO/k9V3hjVUF/zwW7KBVdSK4gIt/bzwS9MbR5qob+F5jusZsb0YQK2A==",
-      "dev": true
-    },
-    "node_modules/concat-stream": {
```

Comment:
Have you not done npm install while setting up the project ??

Reply:
I'm using node version `14.x` and I can go with higher version of node but for using `yarn` the Node version should be `14.x`